### PR TITLE
Modified CMakeLists to support 3dmouse + added custom .cmake file for…

### PR DESCRIPTION
**Pull request by jagazdik**
… 3dmouse - CMakeLists modified and new .cmake file

### CODE REVIEW

File: cmake/Find3DMOUSE.cmake [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Find 3DMOUSE headers and libraries
+#
+#  3DMOUSE_INCLUDE_DIRS - where to find si.h, etc.
+#  3DMOUSE_LIBRARIES    - List of libraries when using 3DMOUSE.
+#  3DMOUSE_FOUND        - True if 3DMOUSE found.
+#=============================================================================
+if( WIN32 )
```
Comment (kapecp) [inline cmake/Find3DMOUSE.cmake]: treba aj ostatne platformy (Linux, OSX)

**Comment by kapecp** [comment]
treba najprv overit funkcnost vytvorenim core class pre 3d mys, ktora bude includovat headre a linkovat sa na libky z SDK pre 3d mys

**Comment by jagazdik** [comment]
odstranil som tu WIN32 hlavicku, s Misom to otestujeme ked budeme mat core
class

**Comment by cimox** [comment]
Merged.

**Pull request merged**